DEPRECATED.md

```diff
@@ -23,6 +23,8 @@ A logged warning is expected for each deprecated item that is in deprecation win
   is deprecated. Please use the new `upgrade_configs` in the
   [HttpConnectionManager](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto)
   instead.
+* Use of `response_headers_to_*` and `request_headers_to_add` are deprecated at the `RouteAction`
+  level. Please use the configuration options at the `Route` level.
```

Comment:
In addition to this, please add `[deprecated = true]` annotation to the relevant fields inside `RouteAction`. e.g. for https://github.com/envoyproxy/envoy/blob/598f5c9433707019169669c787d606ff0b07e9d4/api/envoy/api/v2/route/route.proto#L502 make it `repeated core.HeaderValueOption request_headers_to_add = 12 [deprecated = true];` like what we have [here](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/ratelimit/v2/rls.proto#L45) for `use_data_plane_proto` in rls.proto